packages/sdk/src/transferContractOwnership/README.md

```diff
@@ -0,0 +1,55 @@
+[//]: # `{ "title": "transferContractOwnership", "order": 0.4 }`
+
+# Transfer Contract Ownership
+
+Transfers ownership of a given contractId to an intended ownerId, must be called by current owner
+
+**As with all new SDK api methods, this call should be wrapped in [execute](../#execute) and passed a signing method
+
+## transferContractOwnership(args: TransferContractOwnershipArgs): NearContractCall
+
+`transferContractOwnership` takes a single argument of type `TransferContractOwnershipArgs`
+
+```typescript
+export type TransferContractOwnershipArgs = {
+    //id of the contract for which the ownership will be transfered
+    contractId: string;
+    //intended owner of the contract
+    nextOwner: string;
+    options?: {
+        //keep the current minters, defaults to true if options aren't given
+      keepMinters: boolean;
+    };
+  };
+```
+
+Example usage of transferContractOwnership method in a hypothetical React component:
+{% code title="transferContractOwnershipUI.ts" overflow="wrap" lineNumbers="true" %}
+
+```typescript
+import { useState } from 'react';
+import { useWallet } from '@mintbase-js/react';
+import { execute, transferContractOwnership } from '@mintbase-js/sdk';
+
+
+export const transferContractOwnershipUI = ({ nextOwner, contractId }:any) => {
```

Comment:
Missing space after colon. Does ESLint not work for code embedded in MD?

Reply:
It does not unfortunately, I think after this effort we'll want to take two full passes at documentation.. one right before alpha release prior to merging pr in gitbook-docs repo and then one before we release beta

Reply:
Added it in, lets review this more thoroughly before release like you said